tests/memberclient.py

```diff
@@ -38,8 +38,12 @@ def test_set_recovery_threshold(network, args, recovery_threshold=None):
 def test_add_member(network, args):
     primary, _ = network.find_primary()
 
+    member_data = {"name": "test_added_member", "roles": ["OWNER", "CONTRIBUTOR"]}
```

Comment:
Super minor: When reading this, I genuinely thought that `OWNER` and `CONTRIBUTOR` were actual roles with specific semantics in CCF. For anyone (e.g. CCF operators) reading this test in isolation (e.g. to find examples of governance operations), could we replace these with dummy strings so that it's clear that these roles don't actually exist?